pom.xml

```diff
@@ -114,8 +114,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.3</version>
         <configuration>
-          <source>1.6</source>
```

Comment:
(unrelated chnage)

Reply:
Yep, don't need it after all.

Reply:
Will upgrade the source and target source levels in a separate PR, once this is merged.
Unfortunately GitHub doesn't support chained changes :-(

Reply:
Yes, it is probably time to do so. I've tried to keep it low because I remember there were some users on OSX, which used IntelliJ with a very old JRE because no newer version was available, but I don't think this is the case anymore.